x/gov/endblocker_test.go

```diff
@@ -166,6 +167,7 @@ func TestTickPassedVotingPeriod(t *testing.T) {
 	require.False(t, depositsIterator.Valid())
 	depositsIterator.Close()
 	require.Equal(t, StatusRejected, keeper.GetProposal(ctx, proposalID).GetStatus())
+	require.True(t, TallyResultEqual(keeper.GetProposal(ctx, proposalID).GetTallyResult(), EmptyTallyResult()))
```

Comment:
Can we add a quick testcase for a non-empty tally result?

Reply:
Added in following test, `TestSlashing`